cfme/infrastructure/provider/virtualcenter.py

```diff
@@ -56,7 +56,8 @@ def from_config(cls, prov_config, prov_key, appliance=None):
                    appliance=appliance)
 
     def as_fill_value(self):
-        return {'name': self.name, 'prov_type': 'VMware vCenter'}
+        return {'name': self.name,
+                'prov_type': 'VMware vCenter'}
```

Comment:
this should return a string with provider name as as_fill_value is designed to be applied if you want to eg. select the provider somewhere else, then you don't have to remember if it is name or another field. In order to represent the filling values for filling the model's form, you need to make a custom method probably if you want it like this.